### PR TITLE
[Marketplace] Add App Details Page Adjustments and Next Steps page

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 1]
```diff
@@ -116,7 +115,7 @@
 												"definition": {
 													"fragment": {
 														"key": "image",
-														"siteKey": "[$GROUP_KEY$]"
+														"siteKey": "Marketplace"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:118]: Please make the replacement
```suggestion
														"siteKey": "[$GROUP_KEY$]"
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 2]
```diff
@@ -208,14 +207,17 @@
 													{
 														"definition": {
 															"fragment": {
-																"key": "BASIC_COMPONENT-heading"
+																"key": "text",
+																"siteKey": "Marketplace"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:211]: Please make the replacement
```suggestion
																"siteKey": "[$GROUP_KEY$]"
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 3]
```diff
@@ -277,7 +283,7 @@
 																			"value": {
 																				"html": {
 																					"mapping": {
-																						"fieldKey": "ddmTemplate__ddmTemplate_8811548",
+																						"fieldKey": "AssetVocabulary_47184",
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:286]: I think this needs to be replaced. Please, check with Nilton about which replacement you can use here.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 4]
```diff
@@ -1462,51 +1326,213 @@
...
+																								"paddingTop": 0,
+																								"widthType": "Fluid"
+																							}
+																						},
+																						"pageElements": [
+																							{
+																								"definition": {
+																									"fragment": {
+																										"key": "icon",
+																										"siteKey": "Marketplace"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:1417]: Please make the replacement
```suggestion
																										"siteKey": "[$GROUP_KEY$]"
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json [hunk 5]
```diff
@@ -1462,51 +1326,213 @@
...
+																						}
+																					],
+																					"size": 12
+																				},
+																				"pageElements": [
+																					{
+																						"definition": {
+																							"fragment": {
+																								"key": "icon",
+																								"siteKey": "Marketplace"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json:1499]: Please meke the replacement.
```suggestion
																								"siteKey": "[$GROUP_KEY$]"
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some sugestions.

**Review by claricebrasil — APPROVED** [review]

**Comment by joanagrego** [comment]
ci:test:sf

**Review by silasbsi — COMMENTED** [review]
Good job. :heavy_check_mark:

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by joanagrego** [comment]
ci:reevaluate:1257401_1630

**Comment by joanagrego** [comment]
ci:reevaluate:1257401_1630

**Pull request closed without merging**